transport/httptransport/http_transport.go

```diff
@@ -107,6 +118,7 @@ func (h *httpTransport) Execute(ctx context.Context, transportInfo []byte, dealI
 	if fileSize > dealInfo.DealSize {
 		return nil, fmt.Errorf("deal size=%d but file size=%d", dealInfo.DealSize, fileSize)
 	}
+	h.dl.Infow(duuid, fmt.Sprintf(transportLog, "got existing file size"), "file size", fileSize, "deal size", dealInfo.DealSize)
```

Comment:
Instead of doing `fmt.Sprintf(transportLog, "got existing file size")` for each log line in the file, can we get add a method to the DealLogger that returns a subsystem logger that does it automatically, eg:
```
log := DealLogger.Subsystem("http-transport")
log.Infow(...)
```